src/interfaces/challenge.ts

```diff
@@ -0,0 +1,18 @@
+// export enum StatusState {
+//   Progress = 0,
+//   Success,
+//   Failure
+// }
+
+export interface IChallenge {
+  id: number;
+  state: string;
+  startDate: string;
+  tasksOrder: string;
+  tasksStatus: string;
+  achievementsStatus: string;
+}
+
+export type startNewChallengeType = {
```

Comment:
Should be a function-type.
It will get 4 attributes, with specific types and the 2 default values in the end. 

`type startNewChallenge = (attr....) => IChallenge;`